rasr/config.py

```diff
@@ -338,3 +342,34 @@ def __init__(self, string, hidden=None):
 
     def __str__(self):
         return self.string
+
+
+class WriteRasrConfigJob(Job, RasrCommand):
```

Comment:
```suggestion
class WriteRasrConfigJob(RasrCommand, Job):
```
Its the other way round in AdvancedTreesearchJob. Does this make a difference?

Reply:
No it does not, but we can still do it for consistency.